### DISCUSSION TITLE
Builde and Deploy failed

### DISCUSSION THREAD

**Discussion by rishitachail** [microsoft/SATechnicalOnboarding#130] [Questions]
![Screenshot 2024-05-01 150343](https://github.com/microsoft/SATechnicalOnboarding/assets/108628119/ccca6f5a-f0da-44c2-a5f7-7260d08394d7)

**Comment by AnthonyByansi** [comment]
Hello @rishitachail 
Please review the following section of the technical onboarding instructions; it contains a solution to your query: 

[Deploy your static web app from a GitHub repository](https://github.com/microsoft/SATechnicalOnboarding/blob/main/technical-onboarding-instructions.md#optional-creating-an-azure-static-web-app-from-an-existing-github-respository)
Should you encounter any issues, please feel free to reach out through Teams. I am always here to assist!
